tools/cibadmin.c

```diff
@@ -15,9 +15,7 @@
 #include <crm/common/ipc.h>
 #include <crm/cib/internal.h>
 
-#define local_barf(...) \
-  do { (void)(fprintf(stderr, __VA_ARGS__) > 0 && fflush(stderr)); } while (0)
-#define local_const_barf(_s) local_barf("%s\n", _s);
+#include <pcmki/pcmki_acl.h>
```

Comment:
We've designed it so that pacemaker-internal.h includes everything in pcmki, so code only has to include that one file to get all internal APIs